data/operators/amenity/school.json

```diff
@@ -13287,9 +13287,19 @@
       "displayName": "Kementerian Pendidikan Malaysia",
       "id": "kementerianpendidikanmalaysia-015093",
       "locationSet": {"include": ["my"]},
-      "tags": {
-        "amenity": "school",
-        "operator": "Kementerian Pendidikan Malaysia"
+      "matchNames": ["malaysia ministry of education", "ministry of education malaysia", "moe malaysia"],
+      "tags": {
+        "amenity": "school",
+        "operator": "Kementerian Pendidikan Malaysia",
+        "operator:en": "Ministry of Education",
+        "operator:ms": "Kementerian Pendidikan Malaysia",
```

Comment:
should the be `ms`? or `my`?

Reply:
"ms" there is for Malay.

Reply:
OK, I don't usually do multilingual ones and the difference between the country code stood out to me. But otherwise seems fine.